source/mainnet/docs/guides/overview-desktop.rst

```diff
@@ -22,15 +22,15 @@ Before you start, make sure you’ve completed the following steps:
 
 .. dropdown:: Step 1: Set up the node
 
-    The Desktop Wallet must be connected to a running **trusted** node on the Concordium blockchain. The node enables the Desktop Wallet to interact with the Concordium blockchain to receive updates and submit transactions. A default Virtual Hive node (concordiumwalletnode.com) is provided. But you can run a node yourself using :ref:`Windows<run-node-windows>`, :ref:`macOS <run-node-macos>`, and :ref:`with Docker <run-a-node>` or :ref:`a Debian package <run-node-ubuntu>` on Linux. You can also have a third-party provider run a node for you.
+    The Desktop Wallet must be connected to a running **trusted** node on the Concordium blockchain. The node enables the Desktop Wallet to interact with the Concordium blockchain to receive updates and submit transactions. A default ``grpc.mainnet.concordium.software`` is provided. But you can run a node yourself using :ref:`Windows<run-node-windows>`, :ref:`macOS <run-node-macos>`, and :ref:`with Docker <run-a-node>` or :ref:`a Debian package <run-node-ubuntu>` on Linux. You can also have a third-party provider run a node for you.
 
 .. dropdown:: Step 2: Install the Desktop Wallet
 
     #. Download the latest version of the Desktop Wallet for :ref:`Mainnet<downloads-desktop-wallet>` or :ref:`Testnet<downloads-desktop-wallet-testnet>`.
 
     #. Open the Desktop Wallet and create a password that contains at least 6 characters. Keep the password safe. You’ll need it to sign into the Desktop Wallet again.
 
-    #. If this is the first time you're opening the Desktop Wallet, you're asked to connect to a node. If you don't see this message, go to **Settings**, and then select **Node settings**. The Virtual Hive node (concordiumwalletnode.com) is inserted by default, but you can change this to any other node that you prefer or the node provided by your third-party provider. Enter the **Address** and **Port** of the node you’re running. The address is the network address of the node.
+    #. If this is the first time you're opening the Desktop Wallet, you're asked to connect to a node. If you don't see this message, go to **Settings**, and then select **Node settings**. By default, ``grpc.mainnet.concordium.software`` is inserted, but you can change this to any other node that you prefer or the node provided by your third-party provider. Enter the **Address** and **Port** of the node you’re running. The address is the network address of the node.
```

Comment:
@lassemand can you confirm that this is the endpoint to which the Desktop wallet should connect by default?

Reply:
Short answer: Yes

Long answer:

The issue with external grpc is that we do not provide any uptime/performance guarantees. We would like to eventually have the desktop wallet migrated towards wallet proxy as a backend. For now this is fine.